conveyor/worker_test.go

```diff
@@ -0,0 +1,1008 @@
+/*
+ *    Copyright 2019 Insolar Technologies
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package conveyor
+
+import (
+	"testing"
+	"time"
+
+	"github.com/insolar/insolar/conveyor/adapter"
+	"github.com/insolar/insolar/conveyor/interfaces/constant"
+	"github.com/insolar/insolar/conveyor/interfaces/iadapter"
+	"github.com/insolar/insolar/conveyor/interfaces/islot"
+	"github.com/insolar/insolar/conveyor/interfaces/istatemachine"
+	"github.com/insolar/insolar/conveyor/queue"
+	"github.com/insolar/insolar/core"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+)
+
+var testPulseStates = []constant.PulseState{constant.Future, constant.Present, constant.Past}
+
+func makeSlotAndWorker(pulseState constant.PulseState, pulseNumber core.PulseNumber) (*Slot, worker) {
+	slot := NewSlot(pulseState, pulseNumber, nil)
+	worker := newWorker(slot)
```

Comment:
Where create worker now in conveyer?

Reply:
Now it's not there, since, I think, it's a little bit different task - like, task on integration of this components.